analytic_engine/src/sst/parquet/row_group_pruner.rs

```diff
@@ -14,12 +14,29 @@ use parquet_ext::prune::{
     min_max,
 };
 use snafu::ensure;
+use trace_metric::{MetricsCollector, TraceMetricWhenDrop};
 
 use crate::sst::{
     parquet::meta_data::ParquetFilter,
     reader::error::{OtherNoCause, Result},
 };
 
+#[derive(Debug, Clone, TraceMetricWhenDrop)]
+struct Metrics {
+    #[metric(boolean)]
+    use_bloom_filter: bool,
```

Comment:
No bloom filter is used.... 

May be `row_group_filter`?

Reply:
Actually, all filter is `row_group_filter`. Maybe we can name it `customized_filter`?